flake.nix

```diff
@@ -0,0 +1,74 @@
+{
+    inputs = {
+        # Here you can adjust the IHP version of your project
+        # You can find new releases at https://github.com/digitallyinduced/ihp/releases
+        ihp.url = "github:digitallyinduced/ihp?ref=abb513016b372f9f76b6c95caed66def536a885a";
```

Comment:
So there's no longer `rev`? One has to find the commit `ref` of the release?

Reply:
This will be changed once the new version is tagged. The `rev` is used here only for making it easy to test